app1.py

```diff
@@ -4,6 +4,7 @@
 
 df = pd.read_csv('./final_0427.csv')
 photos = pd.read_csv('./photos_big.csv')
+heart_health = pd.read_csv('./heartDisease_data_9001.csv')

```

Comment:
Same with this one: `heart_health = pd.read_csv('./extraneous_datsets/heartDisease_data_9001.csv')`